src/ActionPanel.scss

```diff
@@ -1,57 +1,91 @@
 @import 'variables';
 
-// TODO: Rework the arrow implementation. For example, it is not quite
-//       centered with respect to the action panel.
-
-.ActionPanel__panel {
-    width: $c2lc-action-panel-width;
+.ActionPanel__background {
+    align-self: center;
     height: $c2lc-action-panel-height;
-    background-color: #F1F1F1;
-    border: 0.1rem solid #939595;
-    border-radius: 3rem;
-    display: flex;
-    justify-content: space-evenly;
-    align-items: center;
-}
-
-.ActionPanel__panel::before {
-    content: '';
     position: absolute;
-    left: 45%;
-    top: 100%;
-    border-left: 0.7rem solid transparent;
-    border-top: 1rem solid #F1F1F1;
-    border-right: 0.7rem solid transparent;
+    top: -0.3rem;
+    width: $c2lc-action-panel-width;
     z-index: 1;
```

Comment:
We have been avoiding using z-index unless it's absolutely necessary, because more z-index we have in our code, harder it will be to maintain. We don't need to use z-index here, if we remove z-index property from the inner container as well. That way, the rendering order will decide which items will appear on top of each other. Let's remove this z-index.

Reply:
Done.